aten/src/ATen/native/cuda/UnaryGeometricAcosKernel.cu

```diff
@@ -5,19 +5,21 @@
 #include <ATen/native/DispatchStub.h>
 #include <ATen/native/TensorIterator.h>
 #include <ATen/native/UnaryOps.h>
+#include <ATen/native/cuda/JitLoops.cuh>
 #include <ATen/native/cuda/Loops.cuh>
 #include <ATen/native/cuda/Math.cuh>
 #include <limits>
 
 namespace at::native {
 
-CONSTEXPR_EXCEPT_WIN_CUDA char acos_name[] = "acos";
+CONSTEXPR_EXCEPT_WIN_CUDA char acos_name[] = "acos_impl";
```

Comment:
The `_impl` is necessary because otherwise it calls `acos` defined in the jiterator headers which fails because the template parameters are different. Same issue exists for all of these.